support-frontend/assets/components/button/upsellButton.tsx

```diff
@@ -0,0 +1,56 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css, ThemeProvider } from '@emotion/react';
+import { from, neutral, space } from '@guardian/source-foundations';
+import {
+	Button,
+	buttonThemeReaderRevenueBrand,
+} from '@guardian/source-react-components';
+
+const button = css`
+	width: 100%;
+	justify-content: space-around;
+	margin: ${space[3]}px 0;
+	color: ${neutral[7]};
```

Comment:
Do we need to set style rules on `@guardian/source-react-components` such as the `<Button>`, I thought these would all be defined by the Button's Theme?

Reply:
These styles were previously being passed to the button using the css prop but should've been passed to the cssOverrides prop instead. With theming I don't think we have a [button theme](https://theguardian.design/2a1e5182b/p/435225-button/t/85cf86) for the button that would match the desired background and text colour, so I'd opted for the closest one with a text colour overwrite